### PR TITLE
Revert "{bio}[GCCcore/13.3.0] rDock v24.03.192"

### PR DESCRIPTION
**Pull request by sassy-crick**
Reverts easybuilders/easybuild-easyconfigs#22012
The user informed me that two directories are missing as the `make install` does not install them.

### CODE REVIEW

**Comment by sassy-crick** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -33,11 +33,16 @@ preinstallopts = 'export PREFIX=%(installdir)s && '
 
 runtest = 'test'
 
-postinstallcmds = ['sed -i "s|/bin/csh -f|/usr/bin/env tcsh -f|" %(installdir)s/bin/make_grid.csh']
+postinstallcmds = [
+    'sed -i "s|/bin/csh -f|/usr/bin/env tcsh|" %(installdir)s/bin/make_grid.csh ; ',
+    'cp -ra %(builddir)s/%(name)s-%(version)s/scripts %(installdir)s ; ',
+    'sed -i "s|/bin/csh -f|/usr/bin/env tcsh|" %(installdir)s/scripts/make_grid.csh ; ',
```
Comment (smoors) [inline easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb:39]: hm, do we now have 2 `make_grid.csh` copies, one in bin/ and one in scripts/ ?
are they the same?

**Comment by sassy-crick** [comment]
@smoors Do you have any idea why the testbuild fails? It works for me with the `4.9.4` version of EasyBuild. Have there been changes in the framework?

**Comment by smoors** [comment]
segmentation fault, which can have many reasons.
let's try again to see if it's consistent

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Comment by sassy-crick** [comment]
Thanks. I was a bit reluctant to do re-submit the test and waste time and energy. Good to see that is working now. :-)

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**